docs/binlog_http_api.md

```diff
@@ -0,0 +1,186 @@
+# TiDB-Binlog HTTP API
+
+## Pump
+
+`PumpIP` is the ip of the Pump server. `8250` is the default port of Pump.
+
+1. Get the current status of Pump
+
+    ```shell
+    curl http://{PumpIP}:8250/status
+    ```
+
+    ```shell
+    $curl http://127.0.0.1:8250/status
+
+   {
+       "status":{
+           "ip-172-16-5-71:8250":{
+               "nodeId":"ip-172-16-5-71:8250",
+               "host":"172.16.5.71:8250",
+               "state":"online",
+               "isAlive":false,
+               "score":0,
+               "label":null,
+               "maxCommitTS":412518831548007863,
+               "updateTS":412518831338292471
+           },
+           "ip-172-16-5-74:8250":{
+               "nodeId":"ip-172-16-5-74:8250",
+               "host":"172.16.5.74:8250",
+               "state":"online",
+               "isAlive":false,
+               "score":0,
+               "label":null,
+               "maxCommitTS":412518831548007869,
+               "updateTS":412518831325184860
+           },
+           "ip-172-16-5-75:8250":{
+               "nodeId":"ip-172-16-5-75:8250",
+               "host":"172.16.5.75:8250",
+               "state":"online",
+               "isAlive":false,
+               "score":0,
+               "label":null,
+               "maxCommitTS":412518831548007875,
+               "updateTS":412518831534899704
+           }
+       },
+       "CommitTS":412518831849473230,
+       "Checkpoint":{
+   
+       },
+       "ErrMsg":""
+   }
+    ```
+
+1. Get all metrics of Pump
```

Comment:
```suggestion
2. Get all metrics of Pump
```

Reply:
`1.` is the list mark in markdown. The markdown renderer will evaluate the correct number for the list items.
you can see: https://github.com/pingcap/tidb-binlog/pull/805/files?short_path=9d85a25#diff-9d85a258cba937911ccf00980402cef8

Reply:
ok, it's fine.